src/kudu/consensus/consensus_queue.cc

```diff
@@ -998,10 +998,41 @@ void PeerMessageQueue::AdvanceMajorityReplicatedWatermarkFlexiRaft(
   }
 }
 
-std::map<std::string, int> PeerMessageQueue::GetDataCommitQuorum() {
-  std::map<std::string, int> data_commit_quorum;
-  // TODO(ritwikyadav) : Populate quorum from cmeta_
-  return data_commit_quorum;
+void PeerMessageQueue::GetDataCommitQuorum(
+    std::map<std::string, int>* data_commit_quorum) const {
+  CHECK(data_commit_quorum);
+  data_commit_quorum->clear();
+
+  // Map to store the number of voters in each region from the active config.
+  std::map<std::string, int> voter_distribution;
```

Comment:
nit: the voter_distribution can be declared after the check

Reply:
Done.